### PR TITLE
Destinations Bigquery, snowflake, redshift: internal CDK refactors for DV2

### PR DESCRIPTION
**Pull request by edgao**

### CODE REVIEW

File: airbyte-integrations/connectors/destination-redshift/build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ plugins {
 airbyteJavaConnector {
     cdkVersionRequired = '0.10.1'
     features = ['db-destinations', 's3-destinations']
-    useLocalCdk = false
+    useLocalCdk = true
```
Comment (gisripa) [inline airbyte-integrations/connectors/destination-redshift/build.gradle:9]: Can we move this changes to a different branch. so dv_cdk_changes is CDK only

Comment (edgao) [reply airbyte-integrations/connectors/destination-redshift/build.gradle:9]: I'd actually prefer to keep it in to trigger redshift integration tests against the updated cdk code - generally I think it's good practice to run at least one connector's CI for every CDK change

**Review by gisripa — APPROVED** [review]

**Comment by edgao** [comment]
republishing cdk after pulling in master - someone else merged 0.10.4 while I was waiting >.>

**Pull request merged**